docs/source/monitoring/metrics.mdx

```diff
@@ -53,20 +55,17 @@ version in the [`ApolloClient` constructor](/react/api/apollo-client/#the-apollo
 
 #### Using custom headers
 
-For more advanced cases, or to use headers other than the default headers, pass a `generateClientInfo` function into the [usage reporting plugin](../api/plugin/usage-reporting/):
+For more advanced cases, or to use headers other than the default headers, pass a `generateClientInfo` function into the [usage reporting plugin](/apollo-server/api/plugin/usage-reporting/):
+
+<MultiCodeBlock>
 
-```js {10-25}
-const { ApolloServer } = require("apollo-server");
-const {
-  ApolloServerPluginUsageReporting,
-  ApolloServerPluginLandingPageLocalDefault,
-} = require('apollo-server-core');
+```ts {10-25}
```

Comment:
I think these line numbers are incorrect now (and maybe we should switch to the //highlight fencing instead

Reply:
Fixed this with `//highlight` fencing :)